hw/ip/alert_handler/data/alert_handler.hjson

```diff
@@ -389,7 +488,7 @@
         }
       ]
     },
-    { name:     "CLASSA_REGWEN",
+    { name:     "CLASSA_CLR_REGWEN",
```

Comment:
this just occurred to me, did we discuss way back then whether the ability to `clear` should be gradually removed?
Ie, we can remove it if we're in phase0, but if we move into phase1, we're going to escalate without stopping? 

It just made me wonder..if we map phase0 to `NMI`, it makes sense to allow clear.  But once we escalate beyond that if no one comes in time, it may not make sense anymore.  

Anyhow, this is just a thought and not really related to this PR.  If it makes sense we can consider it way in the future.

Reply:
No I think the decision whether to allow clearing or not was always pretty black and white.
Either you allow it, or you don't.

We could technically add more knobs here that map to the phases if needed.

Reply:
it's something to consider. Because i can imagine during an NMI phase to allow clearing, but disallow if it goes beyond that.
Not critical though.  It's something I would probably file an issue on for future discussions.

Reply:
It wouldn't be hard to implement. A slight CSR change would work: either make this a 2bit threshold, or provide one bit per phase.

Reply:
naw let's not change it yet. I'm not 100% convinced of the utility. It's just something that came to mind and could be good to think about later as the software guys sort through how they want to use it.

Reply:
sounds good!